src/main/java/com/uid2/optout/util/HttpMethod.java

```diff
@@ -0,0 +1,12 @@
+package com.uid2.optout.util;
+
+public enum HttpMethod {
+    GET, POST;
+
+    public static HttpMethod fromString(String method) {
```

Comment:
There already is an enum for this https://docs.oracle.com/javame/8.0/api/httpclient/api/com/oracle/httpclient/HttpMethod.html

Reply:
Done